security/pkg/k8s/chiron/controller.go

```diff
@@ -136,7 +134,7 @@ func NewWebhookController(gracePeriodRatio float32, minGracePeriod time.Duration
 	if err != nil {
 		return nil, err
 	}
-	if len(dnsNames) == 0 {
+	if len(serviceNamespaces) == 0 {
```

Comment:
Although `len(dnsNames)` is equal to `len(serviceNamespaces)`, I think it's better to use `len(serviceNamespaces)`. Because if length is not 0,  we need to build `namespace` `List-Watch`